platforms/core-runtime/build-operations/build.gradle.kts

```diff
@@ -10,6 +10,7 @@ gradlebuildJava.usedInWorkers()
 dependencies {
     api(libs.jsr305)
     api(projects.stdlibJavaExtensions)
+    api(projects.time)
```

Comment:
:x: Doing this without publishing the `:time` subproject as a library will make it impossible for the Maven cache extension to consume our libraries.